File: Orbit.Domain/DTOs/Calendar/AddCalendarEventRequest.cs
namespace Orbit.Domain.DTOs.Calendar
{
    public class AddCalendarEventRequest
    {
        public required string EventName { get; set; }
        public required string EventLocation { get; set; }
        public required string Description { get; set; }
        public required DateTime StartTime { get; set; }
        public required DateTime EndTime { get; set; }
        public required bool IsAllDay { get; set; }
        public required int CalendarEventTypeId { get; set; }
        public string? RecurrenceRule { get; set; }
    }
}

File: orbit.app/interfaces/api/calendar/AddCalendarEventRequest.ts
export interface AddCalendarEventRequest {
  eventName: string
  eventLocation: string
  description: string
  startTime: string // ISO date string
  endTime: string // ISO date string
  isAllDay: boolean
  calendarEventTypeId: number
  recurrenceRule?: string | null
  documentId: number | null
}
